niimpy/test_screen.py

```diff
@@ -27,7 +27,7 @@ def test_screen_duration():
 
 @pytest.fixture
 def screen1():
-    return niimpy.read_csv(io.StringIO("""\
+    return niimpy.read_csv_string("""\
```

Comment:
This is a text fixture, which is a property of pytest (the testing framework): the return of `screen1` automatically gets passed to any test function that has a `screen1` argument - a thing that helps to make tests.  I wouldn't worry about learning all about fixtures (unless you want to), but see them used and then learn.

In this case, I can have some small test data embedded straight in the test file.  I found this more convenient for development than having it stored separately, especially since I needed to carefully examine the data to see what the expected results are.  If you look above, you can see how I use `niimpy.sampledata.TEST_SCREEN_1` in a different test.